file_store/src/cli/dump.rs

```diff
@@ -392,6 +394,31 @@ impl Cmd {
                         "coverage": coverage.coverage,
                     }))?;
                 }
+                FileType::UniqueConnectionsReport => {
+                    let report = UniqueConnectionsIngestReportV1::decode(msg)?;
+                    let req = UniqueConnectionReq::try_from(report.report.unwrap())?;
+                    print_json(&json!({
```

Comment:
nit, do you need to do a selective print ? you could just do `print_json(&req)?;`

Reply:
I think we just want to not have the signature